pkg/tekton/pipeline_info.go

```diff
@@ -151,7 +151,7 @@ func CreatePipelineRunInfo(prName string, podList *corev1.PodList, ps *v1.Pipeli
 		CreatedTime: pr.CreationTimestamp.Time,
 	}
 
-	if strings.HasPrefix(pr.Name, "metapipeline-") {
+	if strings.HasPrefix(pr.Name, MetaPipeline.String() + "-") {
```

Comment:
+1, definitely a bug. Any idea how this would manifest itself in the controller at least seemingly not making any progress?

Reply:
Trying to figure that out now. =) https://github.com/jenkins-x/jx/blob/e820d616812cc33dd41859c8fd7e4489f809f017/pkg/cmd/controller/controller_build.go#L1066 is the next line I'm sure should be logged after https://github.com/jenkins-x/jx/blob/e820d616812cc33dd41859c8fd7e4489f809f017/pkg/cmd/controller/controller_build.go#L729, the last line we do see, so something must be going wrong between those.

Reply:
Shouldn't you add the pipeline type also as a label into the PipelienRun? Encoding it into the name seems a bit fragile to me.